src/LibraryManager/Resources/Text.resx

```diff
@@ -274,4 +274,7 @@
   <data name="JsDelivrProviderHintText" xml:space="preserve">
     <value>Example: &lt;library&gt;@&lt;version&gt;</value>
   </data>
+  <data name="DownloadingFile" xml:space="preserve">
+    <value>Downloading file {0}...</value>
```

Comment:
Unicode … char instead?

Reply:
I'm going to resolve this for now, but will try to follow up with someone for better guidance and what we should do here.